Gems/ONNX/Code/Source/Clients/Model.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Component/Component.h>
+#include <AzCore/Component/TickBus.h>
+#include <AzCore/Debug/Timer.h>
+#include <AzFramework/StringFunc/StringFunc.h>
+#include <ONNX/ONNXBus.h>
+#include <onnxruntime_cxx_api.h>
+
+#include "upng/upng.h"
+
+#include <algorithm>
+#include <array>
+#include <filesystem>
+#include <string>
+
+namespace ONNX
+{
+    //! Generic ONNX model which can be used to create an inference session and run inferences.
+    class Model
+    {
+    public:
+        Model()
+        {
+        }
```

Comment:
Nit: Can be changed to `Model() = default;`

Reply:
Committed